app/assets/stylesheets/orders.scss

```diff
@@ -0,0 +1,3 @@
+.card.mb-3 {
```

Comment:
again. I understand that it is orders scss, but it applies to every page. Add some class like order-card